src/test/java/com/pusher/client/channel/impl/PresenceChannelImplTest.java

```diff
@@ -60,6 +60,14 @@ public void testReturnsCorrectSubscribeMessageWhenNumericId() {
                 + AUTH_RESPONSE_NUMERIC_ID + "}}", message);
     }
 
+    @Test
+    public void testStoresCorrectUser() {
+        channel.toSubscribeMessage();
```

Comment:
I don't think it's nice that the user information is added to the presence channel as a side effect of this method call. To me, the name of this method implies there wouldn't be any side effects. But I don't think it needs fixing in this PR as this seems to have been the behaviour before these changes. Might be something to fix in the future though.